lib/resty/lmdb.lua

```diff
@@ -47,4 +72,23 @@ do
 end
 
 
+function _M.get_env_info()
+    local env_status = ffi_new("ngx_lua_resty_lmdb_operation_t[1]")
```

Comment:
We can allocate `env_status` in module level.

Reply:
`ngx_lua_resty_lmdb_operation_t` is a wrong name.

Reply:
I don't want to allocate this struct in C level, because I do not want to manage memory in lua side,  lua gc can manage ffi memory

Reply:
like this: https://github.com/openresty/lua-resty-core/blob/0173d96c9eb77b513b989b765716fd2498f09dd9/lib/ngx/ssl.lua#L188-L190

Reply:
It's not likely this, lmdb works on the main module(not the HTTP module or stream module), I think we do not need to access these `r->pool`,  although it can be done by ffi function set argument `pool`, but using luajit manager this it's OKay

Reply:
@dndx , What's your opinion?

Reply:
Allocating from Lua is fine, this is not a hotpath and makes management of memory easier.